### PR TITLE
docs: start doc-comments with a single sentence summary

### PR DESCRIPTION
**Pull request by daadu**
https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary

### CODE REVIEW

File: lib/src/scaffold.dart [hunk 1]
```diff
@@ -143,8 +150,9 @@ class BackdropScaffold extends StatefulWidget {
   /// If null, the color is handled automatically according to the theme.
   final Color? frontLayerBackgroundColor;
 
-  /// Fraction of the available height the front layer will occupy,
-  /// when active (back layer concealed).  Clamped to (0, 1).
+  /// Fraction of available height the [frontLayer] will occupy when active (back layer concealed).
```
Comment (daadu) [inline lib/src/scaffold.dart:153]: can we make this a bit shorted? any idea? by few words - crossing recommend line width

Comment (daadu) [reply lib/src/scaffold.dart:153]: @WieFel

Comment (WieFel) [reply lib/src/scaffold.dart:153]: For example:
Fraction of screen height the active (visible) [frontLayer] will occupy.

Comment (daadu) [reply lib/src/scaffold.dart:153]: let's keep it as it is, just break the line.

Comment (daadu) [reply lib/src/scaffold.dart]: done with - 710ead8

**Comment by daadu** [comment]
Ready for review.

**Review by WieFel — APPROVED** [review]

**Pull request merged**